index.html

```diff
@@ -38,8 +38,20 @@
         margin: 0 auto;
       }
     </style>
+    
   </head>
   <body>
+    <link rel="stylesheet" href="GoingAboveAndBeyond.css">
+    <h1>The Gelatinous Blog!</h1>
+    <img src="gelatiousblog.png" alt="blog-logo"/>
+    
+    <nav>
+      <ul>
+        <li><a href="Maps.html">Maps</a></li>
+        <li><a href="Characters.html">Characters</a></li>
+        <li><a href="Players.html">Players</a></li>
+      </ul>
+    </nav>
```

Comment:
This is missing the `header` element! 

Please review the tutorial exercises to find known good code samples on how to correctly implement this.